app/models/user.rb

```diff
@@ -68,6 +68,7 @@ class User < ApplicationRecord
   include Users::Presentable
   include Users::Stripeable
   include Eventable
+  include FullNameSplitter
```

Comment:
This may conflict with methods `models/concerns/users/presentable.rb`. We should probably just get rid of any similar methods in the concern.

Reply:
I don't think it conflicts actually. Only the full_name= is being introduced with the gem